### PR TITLE
Release announcement for 0.2.1

### PR DESCRIPTION
**Pull request by betarelease**

### CODE REVIEW

File: blog/2022-11-30-pyrsia-0.2.1-released.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+* Better CLI experience - easy to configure
+* MacOSX installer is able to start the service automatically - better experience for getting started.
+* Now you are able to automatically request builds on Pyrsia if an artifact does not exist on the network.
+* Improved and updated documentation
+
+## Integration tests and Staging environment
+
+We invested a lot of time in this release to improve our testing etiquette this release.
+We now have a set of [integration tests](https://github.com/pyrsia/pyrsia-integration-tests) that we rely on to ensure that the network is stable and the apis we offer continue to remain stable.
```
Comment (erwin1) [inline blog/2022-11-30-pyrsia-0.2.1-released.md:22]: 'apis' should be 'APIs'? 
However, I would maybe rephrase to:
```suggestion
We now have a set of [integration tests](https://github.com/pyrsia/pyrsia-integration-tests) that we rely on to ensure that the network is stable and the functionality Pyrsia offers, continues to remain stable.
```

File: blog/2022-11-30-pyrsia-0.2.1-released.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+We did not have a dedicated infrastructure to test before going to production. So as invested in a staging that allows us to test in a controlled environment before we release to production. This environment also allows us to experiment with our infrastructure as well before we make those changes into production.
+
+More testing is better!
+
+Please download our installers, try out Pyrsia with docker and let us know if you are looking for something we do not yet support.
```
Comment (erwin1) [inline blog/2022-11-30-pyrsia-0.2.1-released.md:27]:
```suggestion
Please [download our installers](https://pyrsia.io/docs/tutorials/quick-installation/), try out Pyrsia with Docker and [let us know](https://pyrsia.io/docs/community/get_involved/) if you are looking for something we do not yet support.
```

**Review by fishseabowl — APPROVED** [review]
